packages/xdl/src/credentials/IosCredentials.ts

```diff
@@ -100,19 +102,38 @@ async function getExistingUserCredentials(
   appleTeamId: string,
   type: string
 ): Promise<CredsList> {
-  const { err, certs } = await Api.callMethodAsync('getExistingUserCredentials', [], 'post', {
-    username,
-    appleTeamId,
-    type,
-  });
+  let certs;
```

Comment:
looks like `getExistingUserCredentials` is only used by `getExistingPushKeys` and `getExistingDistCerts` which are not used anymore so u might be able to get rid of all of them

Reply:
Nice catch.